layouts/partials/helpers/text-color.html

```diff
@@ -0,0 +1,13 @@
+{{- $bg := .self.Scratch.Get "bg" }}
+{{- $text_muted := cond (.muted | default false) "text-muted " "" -}}
+{{- $text_dark := cond (isset . "dark") (cond (eq .dark "") "" (printf "text-%s" .dark)) "text-dark" -}}
+{{- $text_light := cond (isset . "light") (cond (eq .light "") "" (printf "text-%s" .light)) "text-light" -}}
```

Comment:
What's the idea behind setting .dark and .light? Shouldn't that be something that the helper figures out from the background?
With a manual overwrite say:
```
{{- partial "helpers/text-color.html" (dict "self" $self "overwrite" "primary") -}}
```

Where all text-colors would be set to primary with the exception of a primary background.
With a primary background it would be an dark for example. Does that make sense?